exporter/sumologicexporter/filter.go

```diff
@@ -44,30 +43,32 @@ func newFilter(flds []string) (filter, error) {
 
 // filterIn returns fields which match at least one of the filter regexes
 func (f *filter) filterIn(attributes pdata.AttributeMap) fields {
-	returnValue := make(fields)
+	returnValue := pdata.NewAttributeMap()
```

Comment:
it is small inconsistency, it will be good to keep the same style on variable naming in the whole package, so I suggest using either `returnValue` or `rv`